lib/bn.js

```diff
@@ -731,6 +731,45 @@ BN.prototype.uxor = function uxor(num) {
 };
 
 
+// Not ``this`` with ``width`` bitwidth
+BN.prototype.inotn = function inotn(width) {
+  assert(typeof width === 'number' && width >= 0);
+
+  var bytesNeeded = width / 26;
```

Comment:
I had it there before but it doesn't really make a difference - even in the case of 27/26, 1<1.0384.. == 1<2. Added back to avoid issues with broken JS interpreters.